src/librustdoc/clean/types.rs

```diff
@@ -1718,11 +1716,6 @@ crate struct Enum {
     crate variants_stripped: bool,
 }
 
-#[derive(Clone, Debug)]
-crate struct Variant {
-    crate kind: VariantKind,
-}
-
 #[derive(Clone, Debug)]
 crate enum VariantKind {
```

Comment:
Would it make sense to rename this into `Variant`? `VariantKind` isn't the worst, but it seems a bit odd without it being attached to a `Variant`.

Reply:
Sure, makes sense. I'll change it.